tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -2003,3 +2070,141 @@ def test_model_k_256x256(
         None,
         dilation=dilation_h,
     )
+
+
+@skip_for_grayskull()
```

Comment:
why do we skip gs and bh seems like we would like to test this feature on those arch as well

Reply:
yes, lets not skip grayskull, but we can skip bh for now since that is still in works :)